### PR TITLE
Update polkadot to v0.9.27

### PR DESCRIPTION
**Pull request by Niederb**

### CODE REVIEW

File: polkadot-parachains/integritee-runtime/Cargo.toml [hunk 1]
```diff
@@ -12,87 +12,87 @@ edition = '2021'
...
 
 # integritee pallets
-pallet-claims = { default-features = false, git = "https://github.com/integritee-network/pallets.git", branch = "polkadot-v0.9.26" }
-pallet-sidechain = { default-features = false, git = "https://github.com/integritee-network/pallets.git", branch = "polkadot-v0.9.26" }
-pallet-teerex = { default-features = false, git = "https://github.com/integritee-network/pallets.git", branch = "polkadot-v0.9.26" }
-pallet-teeracle = { default-features = false, git = "https://github.com/integritee-network/pallets.git", branch = "polkadot-v0.9.26" }
+pallet-claims = { default-features = false, git = "https://github.com/integritee-network/pallets.git", branch = "tn/polkadot-v0.9.27" }
+pallet-sidechain = { default-features = false, git = "https://github.com/integritee-network/pallets.git", branch = "tn/polkadot-v0.9.27" }
+pallet-teerex = { default-features = false, git = "https://github.com/integritee-network/pallets.git", branch = "tn/polkadot-v0.9.27" }
+pallet-teeracle = { default-features = false, git = "https://github.com/integritee-network/pallets.git", branch = "tn/polkadot-v0.9.27" }
```
Comment (echevrier) [inline polkadot-parachains/integritee-runtime/Cargo.toml:60]: I assume you will change the branch to polkadot-v0.9.27. Don't merge this PR before.

**Review by echevrier — APPROVED** [review]
According to the changes in cumulus, I think we should bump clap to 1.9.0, serde to 1.0.140, smallvec to 1.9.0. I will do that

**Comment by echevrier** [comment]
This is working. We can launch again the parachain with polkadot-launch. Thanks a lot.

File: polkadot-parachains/integritee-runtime/Cargo.toml (hunk 1 above)
Comment (Niederb) [reply polkadot-parachains/integritee-runtime/Cargo.toml:60]: This is resolved. I think we are ready to merge

**Review by clangenb — APPROVED** [review]
Nice, this looks like a comparatibely smooth update!

**Comment by clangenb** [comment]
Nice that the polkadot-launch works again! :D

**Pull request merged**